### PR TITLE
feat: Creat a growing normalization layer

### PR DESCRIPTION
**Pull request by TheoRudkiewicz**

### CODE REVIEW

**Comment by sylvchev** [comment]
The comments from Copilot regarding [register_buffer](https://docs.pytorch.org/docs/1.1.0/nn.html#torch.nn.Module.register_buffer) are interesting.

**Review by stelladk — APPROVED** [review]
Looks good!
